scene/gui/color_picker.h

```diff
@@ -164,6 +175,8 @@ class ColorPicker : public VBoxContainer {
 	ColorPresetButton *selected_recent_preset = nullptr;
 	Ref<ButtonGroup> preset_group;
 	Ref<ButtonGroup> recent_preset_group;
+	Callable quick_open_callback;
+	Callable palette_saved_callback;
```

Comment:
`_file_quick_selected()` is wrapped in TOOLS_ENABLED, but not the callbacks?
All editor-only methods and fields should be wrapped consistently.